quarkus/runtime/src/main/resources/application.properties

```diff
@@ -30,6 +30,15 @@ quarkus.log.category."io.quarkus.arc.processor.IndexClassLookupUtils".level=off
 quarkus.log.category."io.quarkus.hibernate.orm.deployment.HibernateOrmProcessor".level=warn
 quarkus.log.category."io.quarkus.deployment.steps.ReflectiveHierarchyStep".level=error
 
+# SqlExceptionHelper will log-and-throw error messages.
+# As those messages might later be caught and handled, this is an antipattern so we prevent logging them
+# https://hibernate.zulipchat.com/#narrow/channel/132096-hibernate-user/topic/Feature.20Request.3A.20Disable.20logging.20of.20SqlExceptionHelper.20for
+quarkus.log.category."org.hibernate.engine.jdbc.spi.SqlExceptionHelper".level=off
```

Comment:
Based on [these messages](https://hibernate.zulipchat.com/#narrow/channel/132096-hibernate-user/topic/Feature.20Request.3A.20Disable.20logging.20of.20SqlExceptionHelper.20for/near/515237070), are we sure we want to disable all Hibernate exceptions logging? 

If there is a place where we incorrectly handle an exception it could save us a lot of time investigating it if we have the exception printed in the log.

Don't get me wrong I am all in with minimizing log-and-throw pattern but I like the idea to filter one specific exception that is causing confusion the users based on the report rather than disabling logging for all Hibernate exceptions.

Reply:
In Keycloak, we have the `KeycloakErrorHandler` that is handling this: The exception is either presented to the user and then it is logged in debug mode, or it is logged as an error. 

In a situation that you describe, we could still enable debug logging (at least for the class `KeycloakErrorHandler`).

As this has been raised several times when we write to the DB in the persistent session worker, I'd rather proceed and hide those logs.

Reply:
Ah, ok. I forgot all of the exceptions would end up in the `Handler`. Sorry for confusion.

Reply:
> Ah, ok. I forgot all of the exceptions would end up in the `Handler`.

There might still be corner cases, but most of them should show in the handler.